lib/match-manager.coffee

```diff
@@ -0,0 +1,60 @@
+_ = require 'underscore-plus'
+{CompositeDisposable} = require 'atom'
+
+module.exports =
+class MatchManager
+  smartQuotePairs:
```

Comment:
Sure, that's easy enough to chop out. I wasn't sure if there was a specific purpose or desire to treat smartquotes distinctly. I personally never use them...